src/js/components/plans/preflightResults.tsx

```diff
@@ -125,10 +123,14 @@ export const getErrorInfo = ({
       currentJob.status === CONSTANTS.STATUS.FAILED ||
       canceledPreflight;
     info.failed = Boolean(failed);
-    info.message =
-      preflight && !(currentJob as Preflight).is_valid && !failed
-        ? i18n.t(`${label} has expired; please run it again.`)
-        : `${i18n.t(`${label} encountered`)} ${msg}.`;
+    if (preflight) {
+      info.message =
+        !preflight.is_valid && !failed
+          ? i18n.t('Pre-install validation has expired; please run it again.')
+          : `${i18n.t('Pre-install validation encountered')} ${msg}.`;
+    } else {
+      info.message = `${i18n.t('Installation encountered')} ${msg}.`;
+    }
```

Comment:
@jgerigmeyer wait... shouldn't this include the word "error" before "encountered"?

Reply:
No, this isn't "Installation error encountered..." -- it's "Installation encountered [3 errors]"

Reply:
@jgerigmeyer ah, I see. In that case we should probably be using [interpolation](https://www.i18next.com/translation-function/interpolation) so that the translators have the full context, and so that we can translate into languages that have a different order of subject-verb-object.

Reply:
@davisagli Yes, I think that's true. We're using it a few places, but not consistently -- and I don't think there's a good reason for that.